com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -386,6 +386,12 @@ Path tracing now supports fog absorption. Like SSS, to use this feature, enable
 
 For information on fog in HDRP, see [fog](Override-Fog.md).
 
+#### Materials support
+
+![](Images/Path-traced-materials-Feature.png)
+
+Path tracing supports almost all the rasterization engine [materials](#Material) listed above: it can render Lit, Layered Lit, Unlit, Stacklit, Fabric and AxF materials. Note that objects using non-HDRP materials simply won't appear in the image, when in path tracing mode.
```

Comment:
Path tracing can render the following rasterization engine [Materials](#Material): 

- Lit
- Layered Lit
- Unlit,
- Stacklit
- Fabric
- AxF 

When path tracing is enabled, objects that use non-HDRP Materials won't appear in the final image.